gulpfile.esm.js

```diff
@@ -97,7 +94,7 @@ function cordovaCompile() {
   const compileArgs = platform === "ios" ? "--device" : "";
   let releaseArgs = "";
   if (buildMode === "release" && platform === "android") {
-    releaseArgs = `--release -- --keystore=${KEYSTORE} --storePassword=${STOREPASS} --alias=${KEYALIAS} --password=${KEYPASS}`;
+    releaseArgs = `--release -- --keystore=keystore.p12 --alias=privatekey --storePassword=${androidKeyStorePassword} --password=${androidKeyStorePassword}`;
```

Comment:
Are the store and key passwords the same?

Reply:
I think you can omit the key password if it's the same as the keystore password.